wire-schema/src/main/java/com/squareup/wire/schema/EnumType.java

```diff
@@ -22,7 +22,11 @@
 import java.util.Collection;
 import java.util.Map;
 
+import static com.squareup.wire.schema.Options.ENUM_OPTIONS;
+
 public final class EnumType extends Type {
+  static final ProtoMember ALLOW_ALIAS = ProtoMember.get(ENUM_OPTIONS, "allow_alias");
```

Comment:
Actually we already do in `isDeprecated()` and `isPacked()` for a field, which is good enough for me.